packages/calimero-sdk/src/subscriptions/subscriptions.ts

```diff
@@ -15,7 +15,7 @@ export interface SubscriptionsClient {
 export type NodeEvent = ApplicationEvent;
 
 export interface ApplicationEvent {
-  application_id: ApplicationId;
+  context_id: ContextId;
```

Comment:
can we fix this side now?

Reply:
sure

Reply:
Should i just add rename camelCase ?

Reply:
it's already this on the rust side, I believe

https://github.com/calimero-network/core/blob/83c09bb45dc46d6cd119471a960e70a1e93867a1/crates/primitives/src/events.rs#L13-L15